src/main/java/frc/robot/maps/ColdStart.java

```diff
@@ -187,10 +188,30 @@ public DeepClimbMap getDeepClimbMap() {
         CSSparkMax motor = new CSSparkMax(13);
         SparkMaxConfig config = new SparkMaxConfig();
         config.smartCurrentLimit(30);
+        config.idleMode(IdleMode.kBrake);
         motor.getMotorController().configure(config, ResetMode.kResetSafeParameters, PersistMode.kPersistParameters);
+        motor.setInverted(true);
```

Comment:
Should use the config object to set inverted, if you're already using it for other things.